packages/bindgen/src/realm_js_helpers.h

```diff
@@ -194,6 +194,15 @@ struct Helpers {
     using LoggerFactory = std::function<std::shared_ptr<util::Logger>(util::Logger::Level)>;
     using LogCallback = std::function<void(util::Logger::Level, const std::string& message)>;
     static LoggerFactory make_logger_factory(LogCallback&& logger)
+    {
```

Comment:
@RedBeard0531 I followed your suggestion of creating a `make_logger` method that is being used by the older `make_logger_factory` method. 
I just wanted to ask if it makes sense the way that I've done it.